app/controllers/users/piv_cac_recommended_controller.rb

```diff
@@ -0,0 +1,43 @@
+module Users
+  class PivCacRecommendedController < ApplicationController
+    include TwoFactorAuthenticatableMethods
+    include MfaSetupConcern
+    include SecureHeadersConcern
+    include ReauthenticationRequiredConcern
+
+    before_action :authenticate_user!
+    before_action :confirm_user_authenticated_for_2fa_setup
+    before_action :apply_secure_headers_override
+    before_action :user_email_is_gov_or_mil?
+
+    def show
+      @recommended_presenter = PivCacRecommendedPresenter.new(current_user)
+      analytics.piv_cac_recommended_visited
+    end
+
+    def confirm
+      UpdateUser.new(
+        user: current_user,
+        attributes: { piv_cac_recommended_visited_at: Time.zone.now },
+      ).call
+      analytics.piv_cac_recommended(action: :accepted)
+      set_mfa_selections(['piv_cac'])
+      redirect_to confirmation_path(user_session[:mfa_selections].first)
```

Comment:
Could we use `next_setup_path` here? Or another helper from `MfaSetupConcern` that we're already using to direct to the "next" setup path?

Reply:
I dont think we have another helper for this but might be worth to create one.